haskell-customize.el

```diff
@@ -429,42 +417,48 @@ imports."
   "The path which is considered as project root, this is determined by the
 presence of a *.cabal file or stack.yaml file or something similar.")
 
+(defun haskell-build-type ()
```

Comment:
Just an observation that there are now many instances of haskell-related elisp code that do something like the below, e.g. in `dante`. I wish we could unify them!